File: ios/Classes/CustomTabsPlugin.m
#import "CustomTabsPlugin.h"

@implementation GDBCustomTabsPlugin
+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
  FlutterMethodChannel* channel = [FlutterMethodChannel
      methodChannelWithName:@"com.github.droibit.flutter.plugins.custom_tabs"
            binaryMessenger:[registrar messenger]];
  GDBCustomTabsPlugin* instance = [[GDBCustomTabsPlugin alloc] init];
  [registrar addMethodCallDelegate:instance channel:channel];
}

- (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
  result(FlutterMethodNotImplemented);
}

@end

File: ios/Classes/CustomTabsPlugin.swift
import Flutter
import UIKit

public class CustomTabsPlugin: NSObject, FlutterPlugin {
  public static func register(with registrar: FlutterPluginRegistrar) {
    let channel = FlutterMethodChannel(name: "com.github.droibit.flutter.plugins.custom_tabs", binaryMessenger: registrar.messenger())
    let instance = CustomTabsPlugin()
    registrar.addMethodCallDelegate(instance, channel: channel)
  }

  public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
    result(FlutterMethodNotImplemented)
  }
}
